modules/images-update-global-pull-secret.adoc

```diff
@@ -16,13 +16,6 @@ You can update the global pull secret for your cluster by either replacing the c
 Cluster resources must adjust to the new pull secret, which can temporarily limit the usability of the cluster.
 ====
 
-Updating the global pull secret causes the Machine Config Operator to drain the nodes, apply the change, and uncordon the nodes.
```

Comment:
Maybe instead of fully deleting this, we can just have a comment that says the MCO still rolls out those changes, but no longer drains the nodes if you are on 4.8 or above and that is the only change? (technically 4.7.4 but as noted we dont document based on z-stream)

Reply:
+1. To avoid even 1% confusion let's mention both i.e. `no longer require drain or reboot`

Reply:
How does this sound: 
_As of {product-title} 4.7.4, changes to the global pull secret no longer trigger a node drain or reboot._

Reply:
lgtm. Can we also fix https://github.com/openshift/openshift-docs/pull/36129#issuecomment-915059745 ?

Reply:
@sinnykumari 
I moved the note down to https://github.com/openshift/openshift-docs/blame/main/modules/images-update-global-pull-secret.adoc#L64. It seemed redundant to have the note twice. Does this make sense?

Reply:
yeah, good idea.